core/executor/src/lib.rs

```diff
@@ -414,8 +414,8 @@ impl AxonExecutor {
                 let consensus_config = handle.get_consensus_config().unwrap();
                 Some(consensus_config.max_contract_limit as usize)
             } else {
-                // If the hardfork is not enabled, the limit is set to 0x6000
-                evm_config.create_contract_limit
+                // If the hardfork is not enabled, use default_max_contract_limit()
+                Some(default_max_contract_limit() as usize)
```

Comment:
I do not understand the change. The `evm_config.max_contract_limit` is `Some(0x6000)`.

Reply:
```rs
pub fn default_max_contract_limit() -> u64 { 
    0xc000 
}
```
Note: This is a PR to [forcerelay-dev](https://github.com/axonweb3/axon/tree/forcerelay-dev) branch.